db/re/item_db.txt

```diff
@@ -3786,14 +3786,14 @@
 5670,Aniv_Star_Hat,Aniv Star Hat,4,20,,300,,0,,0,0xFFFFFFFF,63,2,256,,1,1,599,{},{},{}
 5671,Drooping_Morocc_Minion,Drooping Morocc Minion,4,20,,300,,0,,1,0xFFFFFFFF,63,2,256,,70,0,600,{ bonus bSPGainValue,2; bonus3 bAutoSpellWhenHit,"NPC_CRITICALWOUND",2,2; },{},{}
 5672,Southern_Cross,Southern Cross,4,20,,600,,3,,1,0xFFFFFFFF,63,2,256,,50,1,601,{ bonus bInt,1; },{},{}
-5673,Home_Cherry_Blossom,Home Cherry Blossom,4,20,,100,,2,,1,0xFFFFFFFF,63,2,256,,30,1,602,{},{},{}
+5673,Home_Cherry_Blossom,Home Cherry Blossom,4,20,,100,,2,,1,0xFFFFFFFF,63,2,256,,30,1,602,{ bonus bNoCastCancel,0; bonus bVariableCastrate,35; },{},{}
 5674,Pig_Moneybox,Pig Moneybox,4,20,,1000,,3,,0,0xFFFFFFFF,63,2,256,,10,0,603,{},{},{}
-5675,Poring_Letter,Poring Letter,4,20,,100,,0,,0,0xFFFFFFFF,63,2,1,,10,0,604,{ bonus bUnbreakableHelm,1; },{},{}
-5676,Scorpio_Crown,Scorpio Crown,4,20,,300,,3,,0,0xFFFFFFFF,63,2,256,,1,1,605,{},{},{}
-5677,Scorpio_Diadem,Scorpio Diadem,4,20,,300,,3,,0,0xFFFFFFFF,63,2,256,,1,1,606,{ bonus bDex,2; },{},{}
+5675,Poring_Letter,Poring Letter,4,20,,100,,0,,0,0xFFFFFFFF,63,2,1,,10,0,604,{ bonus2 bAddMonsterDropItem,619,10; bonus bUnbreakableHelm,1; },{},{}
+5676,Scorpio_Crown,Scorpio Crown,4,20,,300,,3,,0,0xFFFFFFFF,63,2,256,,1,1,605,{ bonus bAspdRate,3; bonus2 bSubEle,Ele_Water,5; if (getrefine()>6) { bonus3 bAutoSpell,"TK_SEVENWIND",3,20; } if (getrefine()>7) { bonus bAspdRate,2; } if (getrefine()>8) { bonus3 bAutoSpell,"WZ_FROSTNOVA",5,10; } if (getrefine()>9) { bonus bAspdRate,2; bonus bAtk,5; } },{},{}
```

Comment:
too many `getrefine()` redundant., `bAtk` must be `bBaseAtk`, and u miss the `bonus bBaseAtk,5;` as non-refine bonus.
```
{ bonus bAspdRate,3; bonus2 bSubEle,Ele_Water,5; bonus bBaseAtk,5; .@r = getrefine(); if (.@r>6) { bonus3 bAutoSpell,"TK_SEVENWIND",3,20; } if (.@r>7) { bonus bAspdRate,2; } if (.@r>8) { bonus3 bAutoSpell,"WZ_FROSTNOVA",5,10; } if (.@r>9) { bonus bAspdRate,2; bonus bBaseAtk,5; } }
```